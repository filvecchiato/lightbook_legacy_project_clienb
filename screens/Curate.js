import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

// mockup for page
const Curate = ({ navigation }) => {
  return (
    <View>
      <Text>Curated gallery</Text>
    </View>
  );
};

const styles = StyleSheet.create({});

export default Curate;
