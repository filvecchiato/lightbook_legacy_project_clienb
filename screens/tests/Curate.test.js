import React from 'react';
import renderer from 'react-test-renderer';
import Curate from '../Curate';

test('renders correctly', () => {
  const tree = renderer.create(<Curate />).toJSON();
  expect(tree).toMatchSnapshot();
});
