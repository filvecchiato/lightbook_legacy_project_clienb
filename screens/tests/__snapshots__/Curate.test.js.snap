// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text>
    Curated gallery
  </Text>
</View>
`;
